RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1757,7 +1763,7 @@ boolean isFreeMonster(monster mon, location loc)
 	}
 
 	if($locations[Shadow Rift (The Ancient Buried Pyramid), Shadow Rift (The Hidden City), Shadow Rift (The Misspelled Cemetary)] contains loc
-		&& have_effect($effect[shadow affinity]) > 0)
+		&& have_effect($effect[shadow affinity]) > 0 && !in_ag())
```

Comment:
minor - can this be removed due to check at start of function?

Reply:
Probably. I think I had left it in in case I thought I should remove the check at the start especially once it goes out of season but that really only applies if any of those are non-adventure.php areas